hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -120,6 +122,7 @@
     private boolean isRatisEnabled = false;
     private boolean isTracingEnabled = false;
     private Function<Long, Long> indexToTerm = null;
+    private Semaphore availPendingRequestNum;
```

Comment:
This variable name is little confusing. Pending request seems like the request still needs to be processed. We can say unflushed transactions or something.